LibertyDSNP/frequency
# Bug: Fix missing extension warning in api-augment

# Goal
The goal of this PR is to remove the warning from the api-augment package about the `HandlesSignedExtension`

Closes #1456 

# Discussion
- Added `HandlesSignedExtension`
- Updated packages

## Testing
- `make js`
- Create a new package that uses the tgz file that is created
- Use it to run the small script below
- Don't see the error!

```ts
import { options } from '@frequency-chain/api-augment';
import { ApiPromise, WsProvider } from "@polkadot/api";

// const WEBSOCKET_URL = 'wss://0.rpc.frequency.xyz'; // Mainnet
const WEBSOCKET_URL = 'wss://rpc.rococo.frequency.xyz'; // Rococo


const getApi = async () => {
  const api = await ApiPromise.create({
    provider: new WsProvider(WEBSOCKET_URL),
    ...options,
  });
  await api.isReady;
  return api;
}

const main = async () => {
  console.log("Connecting to the API...");
  const api = await getApi();
  const blockNumber = await api.query.system.number();
  console.log({
    blockNumber: blockNumber.toHuman(),
  })
}

main().catch(console.error).finally(process.exit);
```
